unit-tests/src/test/java/org/eclipse/collections/impl/bag/mutable/MutableBagTestCase.java

```diff
@@ -278,12 +278,31 @@ public void addOccurrences()
         MutableBagTestCase.assertBagsEqual(HashBag.newBagWith("1", "2", "2", "2", "2", "2", "2", "3"), bag);
     }
 
+    @Test
```

Comment:
Do we need to assert the instance is the same as well?

Reply:
Yes, that is a good thing to assert as well. I can add it.